app/main.py

```diff
@@ -1,3 +1,77 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+import datetime
+
+from app.customer import Customer
+from app.car import Car
+from app.shop import Shop
+
+
+def shop_trip() -> None:
+    with open("app/config.json", "r") as f:
+        data = json.load(f)
+
+    fuel_price = data["FUEL_PRICE"]
+    customers = []
+    shops = []
+
+    for customer in data["customers"]:
+        customer_instance = Customer(
+            customer["name"],
+            customer["product_cart"],
+            customer["location"],
+            customer["money"],
+            Car(
+                customer["car"]["brand"],
+                customer["car"]["fuel_consumption"]
+            )
```

Comment:
```suggestion
            Car(**customer["car"])
```
you can use unpacking here